src/dev/build/tasks/os_packages/docker_generator/templates/ironbank/hardening_manifest.yaml

```diff
@@ -42,39 +42,47 @@ resources:
       value: 0ce56bde1853fed3e53282505bac65707385275a27816c29712ab04c187aa249797c82c58759b2b36c210d4e2683eda92359d739a8045cb8385c2c34d37cc9e1
 # List of project maintainers
 maintainers:
+  # AppEx Operations Members
+  - email: 'brad.white@elastic.co'
+    name: 'Brad White'
+    username: 'brad.white'
+    cht_member: false
   - email: 'jon@elastic.co'
     name: 'Jonathan Budzenski'
     username: 'jbudz'
     cht_member: false
-  - email: 'brad.white@elastic.co'
-    name: 'Brad White'
-    username: 'brad.white'
+  # AppEx Platform Security Members
+  - email: 'aleh.zasypkin@elastic.co'
+    name: 'Aleh Zasypkin'
+    username: 'azasypkin'
+    cht_member: false
+  - email: 'larry.gregory@elastic.co'
+    name: 'Larry Gregory'
+    username: 'legrego'
+    cht_member: false
+  # InfoSec Members
+  - email: 'abby.zumstein@elastic.co'
+    name: 'Abby Zumstein'
+    username: <TODO>
+    cht_member: false
+  - email: 'arsalan.khan@elastic.co'
+    name: 'Arsalan Khan'
+    username: 'kharsalan'
```

Comment:
@kharsalan please let me know once you have a repo1 username so I can update the PR accordingly.